etc/parameters.yml.dist

```diff
@@ -6,8 +6,7 @@ api:
 
 pim:
     install_path: '/srv/pim'
-    is_docker: true
+    is_docker: false
```

Comment:
As the CI will probably use `is_docker: true`, we should let the value as true, like for the CE client.